src/components/Contentful/Hero.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+
+export default function ContentfulHero({ pageTitle, sectionName, id, image }) {
+  let sectionNameComponent
+  let imageComponent
+
+  if (sectionName)
+    sectionNameComponent = <p className="section-name">{sectionName}</p>
+
+  if (image)
+    imageComponent = (
+      <img className="services-image" alt={image.title} src={image.fixed.src} />
+    )
+
+  return (
+    <div className="contentful-hero" id={id}>
+      <div className="container">
+        <div className="row">
+          <div className="col-5">
+            <p className="breadcrumbs">Home/our services/blah blah</p>
+            <div> {sectionNameComponent}</div>
```

Comment:
Extra whitespace here should be removed, unless necessary? If necessary replace with `&nbsp;`.